packer/netbsd_openbsd.pkr.hcl

```diff
@@ -141,7 +150,7 @@ source "googlecompute" "postgres" {
   source_image_family     = "${local.name}-${var.name}-vanilla-${var.version}"
   source_image_project_id = ["${var.gcp_project}"]
   image_family            = "${local.name}-${var.image_name}-${var.version}"
-  image_name              = "${local.name}-${var.image_name}-${var.image_date}"
+  image_name              = local.image_identity
```

Comment:
Huh, why do we use the version in the image family, but not the name? That seems wrong.

Reply:
I added version to image_name. I think we should kept version in the family but maybe only major version? 

Also, I moved `gcloud compute images add-iam-policy-binding` part from `.cirrus.yml` to `netbsd_openbsd.pkr.hcl` because version is got from pkrvars files, not from task_name and it worked. Should we apply same for other tasks?

Reply:
> I added version to image_name. I think we should kept version in the family but maybe only major version?

Yes, that'd make sense. But let's do that in a separate PR.

> Also, I moved gcloud compute images add-iam-policy-binding part from .cirrus.yml to netbsd_openbsd.pkr.hcl because version is got from pkrvars files, not from task_name and it worked. Should we apply same for other tasks?

Hm. It somehow seems wrong to do the iam-policy stuff inside packer, but I can't really explain why. However, you're right, the names don't match up right now. Either way, could we do that as part of a separate PR?